open_cam.py

```diff
@@ -1,71 +1,88 @@
-from tensorflow.python.keras.models import load_model
 import numpy as np
 import cv2
+# configs
 from config import CAFFE_PROTO_FILE, MODEL_FILE, CONFIDENCE
+# video utils
 import imutils, time
 from imutils.video import VideoStream
+# keras api
+from tensorflow.python.keras.models import load_model
 from tensorflow.keras.applications.mobilenet_v2 import preprocess_input
 from tensorflow.keras.preprocessing.image import img_to_array
 
-
+# load model
 keras_model = load_model('model.h5')
+# facenet
```

Comment:
comments its use not its name dude :)

Reply:
:) no here is comment for definition not where it take action. only declaration

Reply:
i think you can remove this comment, because it's not nescessary